apps/molgenis-components/src/client/client.ts

```diff
@@ -29,18 +32,27 @@ const client: IClient = {
       deleteAllTableData: async (tableName: string) => {
         return deleteAllTableData(tableName, schemaNameCache);
       },
-      fetchSchemaMetaData: async () => {
-        schemaMetaData = await fetchSchemaMetaData(myAxios, schemaNameCache);
+      fetchSchemaMetaData: async (useCache: boolean = true) => {
```

Comment:
I would probably not add this parameter and instead ask users to clear cache (or start a new client).

Motivation, but maybe you can solve this differently, is that I expect that next we will change this to `fetchSchemaMetadata(schemaName?:string)` that is to make it possible to also retrieve other schemas metadata next the current one. Would that still be possible with this change?

Reply:
yes and no, could have fetchSchemaMetaData(options?:  fetchSchemaMetaDataOptions)

fetchSchemaMetaDataOptions {
  schema?:string,
  useCache: boolean
}

Reply:
but we could go with no param func for now , and have user clear cache when needed

Reply:
issue is have with that its thats is not clear form looking at the func name and params whats going on ( need to look at the whole set of functions )